src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -912,7 +918,8 @@ def create_ag_url_path_map_rule(cmd, resource_group_name, application_gateway_na
         name=item_name,
         paths=paths,
         backend_address_pool=SubResource(id=address_pool) if address_pool else default_backend_pool,
-        backend_http_settings=SubResource(id=http_settings) if http_settings else default_http_settings)
+        backend_http_settings=SubResource(id=http_settings) if http_settings else default_http_settings,
+        firewall_policy=SubResource(id=firewall_policy) if firewall_policy else None)
```

Comment:
if cmd.supported_api_version(min_api='2019-09-01'):?

Reply:
You are right. Fixing it.